config/samples/logstash/logstash.yaml

```diff
@@ -4,14 +4,23 @@ metadata:
   name: logstash-sample
 spec:
   count: 3
-  version: 8.6.1
+  version: 8.7.0
   config:
     log.level: info
     api.http.host: "0.0.0.0"
     queue.type: memory
   pipelines:
     - pipeline.id: main
       config.string: input { exec { command => 'uptime' interval => 10 } } output { stdout{} }
+  volumeClaimTemplates:
+    - metadata:
+        name: pq
+      spec:
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 1Gi
```

Comment:
Should we try to catch/validate a change here before the user hits ` Forbidden: updates to statefulset spec for fields other than 'replicas', 'template', 'updateStrategy', 'persistentVolumeClaimRetentionPolicy' and 'minReadySeconds' are forbidden` which does not provide a lot of information about the root cause?

Reply:
I implemented a validation on update: 77add7a645.